static/js/navbar_alerts.js

```diff
@@ -146,11 +160,32 @@ export function initialize() {
             custom_class: "red",
             rendered_alert_content_html: render_configure_email_alert_content(),
         });
+    } else if (
+        page_params.demo_organization_scheduled_deletion_date &&
+        demo_org_days_remaining <= 7
+    ) {
```

Comment:
Can you add a comment here documenting the logic behind this block appearing twice?  This code will be confusing for anyone who doesn't understand our intent around the prioritization of the notices here.

Reply:
It's also possible that we should consider just making the demo organization notice the top priority, and have a "Hide notice" button on it that we expect users to click every time they login to a demo organization (rather than this 2-stage model).

@alya for thoughts on that.

Reply:
Added the comments! :)

Reply:
Making the demo org banner the top priority sounds fine to me.

Reply:
I made it the top priority and also added the "Hide notice" button.